extras/gomock.bzl

```diff
@@ -55,14 +55,14 @@ def _gomock_source_impl(ctx):
         aux_files = []
         for target, pkg in ctx.attr.aux_files.items():
             f = target.files.to_list()[0]
-            aux = ctx.actions.declare_file(paths.join(gopath, "src", pkg, f.basename))
+            aux = ctx.actions.declare_file(paths.join("gopath", "src", pkg, f.basename))
```

Comment:
Good catch! Can you extract the string "gopath" to a variable like `go_path_prefix` and use that in both Line 39 and here? So future readers know whether the string "gopath" is from.

Reply:
Sure, that's a great idea! Updated the PR